### ISSUE TITLE
Downloading MNIST dataset with torchvision gives HTTP Error 503

### ISSUE THREAD

**Issue by omarfoq** [pytorch/vision#3549]
## 🐛 Bug

I am getting a 503 error when  I try to download MNIST dataset with `torchvision` 0.9.0.
This is probably due to the fact that http://yann.lecun.com/exdb/mnist/ is unavailable, so apart from pproviding a new link to the dataset, I don't see how this problem can be solved

## To Reproduce

```
from torchvision.datasets import MNIST

MNIST(".", download=True)
```

cc @pmeier

**Comment by baijumeswani** [comment]
It appears that http://yann.lecun.com/exdb/mnist/ is unavailable at this time. A temporary solution before downloading MNIST:

old version of torchvision:
```
from torchvision import datasets
datasets.MNIST.resources = [
            ('https://ossci-datasets.s3.amazonaws.com/mnist/train-images-idx3-ubyte.gz', 'f68b3c2dcbeaaa9fbdd348bbdeb94873'),
            ('https://ossci-datasets.s3.amazonaws.com/mnist/train-labels-idx1-ubyte.gz', 'd53e105ee54ea40749a09fcbcd1e9432'),
            ('https://ossci-datasets.s3.amazonaws.com/mnist/t10k-images-idx3-ubyte.gz', '9fb629c4189551a2d022fa330f9573f3'),
            ('https://ossci-datasets.s3.amazonaws.com/mnist/t10k-labels-idx1-ubyte.gz', 'ec29112dd5afa0611ce80d1b7f02629c')
        ]
```

new version of torchvision:
```
from torchvision import datasets
datasets.MNIST.mirrors = [
        'https://ossci-datasets.s3.amazonaws.com/mnist/',
    ]
```

**Comment by madhavajay** [comment]
![image](https://user-images.githubusercontent.com/2882739/110881112-839f0000-832b-11eb-83de-5d045e5ec5cc.png)

**Comment by pmeier** [comment]
This should be fixed in #3544. [We will release `torchvision==0.9.1`](https://github.com/pytorch/vision/issues/3546#issuecomment-796829072) soon to get this in a release

**Comment by ghazpar** [comment]
So when can we expect the release of 0.9.1 ?

**Comment by fmassa** [comment]
The release of 0.9.1 is expected to be out in around 10 days

**Comment by ghazpar** [comment]
Ok, good to know. 

But given that the unavailability of the MNIST database will break existing codes, I would have expected an earlier release?

Just my 2c.

**Comment by fmassa** [comment]
We've released torchvision 0.9.1 a few days ago https://github.com/pytorch/vision/releases/tag/v0.9.1
